tools/app/__tests__/components.test.tsx

```diff
@@ -348,3 +349,251 @@ const testProject: Project = {
     },
   ],
 };
+
+const testTree: ParseResult['results'] = [
+  {
+    key: 'USDL-43',
+    path: '/Users/jaakko/dev/cyberismo/unified-sdl/cardroot/USDL-43',
```

Comment:
We cannot have absolute paths in tests to user folders.

Reply:
If this is for testing purposes only (the path can be whatever), then please change "jaakko" -> "test" or similar.

Reply:
Yes, the app doesn't interact with the path in anyway. I don't think the parseResult even has that field. I'll remove it.